README.md

```diff
@@ -1 +1,60 @@
-# javascriptcapstone
\ No newline at end of file
+# javascriptcapstone
+> This project is built based on external APIs. One  [Meals DB API](https://www.themealdb.com/api.php) storing information abot meals and another [Involvement API](https://www.notion.so/microverse/Involvement-API-869e60b5ad104603aa6db59e08150270) to help with page interactions
+
+![homeshot](src/assets/homeshot.png)
+![screenshot](src/assets/commentshot.png)
+
+
+
+## Built With
+
+This project is build with:
+
+- ![HTML5](https://img.shields.io/badge/-HTML5-000000?style=flat&logo=html5&logoColor=ffffff&labelColor=E34F26), ![CSS3](https://img.shields.io/badge/-CSS3-000000?style=flat&logo=css3&logoColor=ffffff&labelColor=1572B6), 
+- ![JavaScript](https://img.shields.io/badge/-JavaScript-000000?style=flat&logo=javascript)
+- [Bootstrap5](https://getbootstrap.com/docs/5.0/)
+
+## Getting Started
```

Comment:
- [ ] Kindly include the live deploy link in the readme.

Reply:
Hello, thanks so much for the review. We had a few hitches with deploying as it isn't reflecting our latest changes. We feel it would be a bit fair if you made this change optional since it is not a project requirement. Thanks

Reply:
Well solved